modules/300-prometheus/monitoring/prometheus-rules/deprecation.yaml

```diff
@@ -30,3 +30,79 @@
           __meta_kubernetes_endpoint_address_target_kind -> __meta_kubernetes_endpointslice_address_target_kind
           __meta_kubernetes_endpoint_address_target_name -> __meta_kubernetes_endpointslice_address_target_name
           ```
+
+- name: d8.grafana.interval_deprecation
+  rules:
+    - alert: GrafanaDashboardPanelIntervalDeprecated
+      expr: |
+        d8_grafana_dashboards_deprecated_interval > 0
+      labels:
+        tier: cluster
+        severity_level: "8"
+      annotations:
+        plk_markup_format: markdown
+        plk_protocol_version: "1"
+        plk_create_group_if_not_exists__d8_grafana_deprecations: "GrafanaDeprecations,tier=cluster,kubernetes=~kubernetes"
+        plk_grouped_by__d8_grafana_deprecations: "GrafanaDeprecations,tier=cluster,kubernetes=~kubernetes"
+        summary: Deprecated Grafana panel interval has found.
+        description: |
+          Before updating to Grafana 10, it's required to rewrite an outdated expressions that uses `$interval_rv`, `interval_sx3` or `interval_sx4` to `$__rate_interval`
+          Deprecated interval found on dashboard panel `{{ $labels.dashboard }}/{{ $labels.panel }}`: `{{ $labels.interval }}`
+
+- name: d8.grafana.alert_rule_deprecation
+  rules:
+    - alert: GrafanaDashboardAlertRulesDeprecated
+      expr: |
+        d8_grafana_dashboards_deprecated_alert_rule > 0
```

Comment:
```suggestion
        sum by (dashboard, panel, alert_rule) (d8_grafana_dashboards_deprecated_alert_rule) > 0
```

Reply:
Fixed + added message about query evaluation to the alert description